src/routes/brands/deleteBrand.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  RawReplyDefaultExpression,
+  RawRequestDefaultExpression,
+  RawServerDefault,
+  RouteHandler,
+  RouteOptions,
+} from "fastify";
+import BrandModel from "../../models/brand";
+import BrandSchema from "../../schemas/Brand.json";
+import { Brand } from "../../types/Brand";
+import BrandRequestParamsSchema from "../../schemas/BrandRequestParams.json";
+import { BrandRequestParams } from "../../types/BrandRequestParams";
+
+type Reply = {};
```

Comment:
Will it return an empty object? 🤔

Reply:
This needs correction. What will this endpoint return? An empty object? 
you added the response type for the error, but for 204, it will return an empty object?